extension/source/QuillPage/components/ViewSecretPhrasePanel.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { browser } from 'webextension-polyfill-ts';
+import Button from '../../components/Button';
+import Range from '../../helpers/Range';
+import QuickColumn from './QuickColumn';
+import QuickRow from './QuickRow';
+
+const ViewSecretPhrasePanel: React.FunctionComponent = () => {
+  const [expanded, setExpanded] = React.useState(false);
+
+  const classes = ['view-secret-phrase-panel'];
+
+  if (expanded) {
+    classes.push('expanded');
+  }
+
+  const secretWords = [
+    'Potato',
+    'Velvet',
+    'Keen',
+    'Water',
+    'Travel',
+    'Pill',
+    'Book',
+    'Photo',
+    'Image',
+    'Space',
+    'Pause',
+    'Power',
+  ];
+
+  return (
+    <div className={classes.join(' ')}>
```

Comment:
```suggestion
    <div className={`view-secret-phrase-panel ${expanded && 'expanded'}`}>
```

An easier way to achieve same effect

Reply:
Hmmm I don't like this, won't the result be `'view-secret-phrase-panel false` if `expanded` is `false`?

Reply:
No, actually 

```ts
true && "string" // "string"
false && "string" // null
```

Reply:
yeah agreed, can do
`${expanded ? 'expanded' : ''}`
however its indeed a common pattern to do  
`${expanded && 'expanded'}`

but I am strictly not in favor of having local variables inside a component which are not changed by state update

Reply:
NA now that there isn't an `expanded` class. (https://github.com/jzaki/bls-wallet/pull/93#discussion_r790387789)